python/lsst/ts/m2/tcp_client.py

```diff
@@ -148,6 +148,9 @@ async def _monitor_msg(self):
             self.log.info("Reader disconnected; closing client")
             await self._basic_close()
 
+        except asyncio.IncompleteReadError:
+            self.log.info("EOF is reached.")
```

Comment:
Consider using `self.log.exception` instead, so we get the traceback.

Reply:
Done in the commit: e99800d. Thanks!